### ISSUE TITLE
Project Is Not Running

### ISSUE THREAD

**Issue by ommvoice** [Coevery/Coevery#66]
Hi Guys,
I downloaded this project and while running it got this error.
After i fixed this IOC/Autofec error,  I found Resource not found on
http://localhost:30321/CoeveryLocal/

Any Suggestion?
Thanks.

Error:

None of the constructors found with 'Autofac.Core.Activators.Reflection.DefaultConstructorFinder' on type 'Coevery.Environment.DefaultCoeveryShell' can be invoked with the available services and parameters:
Cannot resolve parameter 'Coevery.Mvc.Routes.IRoutePublisher routePublisher' of constructor 'Void .ctor(System.Func`1[Autofac.Features.OwnedInstances.Owned`1[Coevery.Environment.ICoeveryShellEvents]], System.Collections.Generic.IEnumerable`1[Coevery.Mvc.Routes.IRouteProvider], System.Collections.Generic.IEnumerable`1[Coevery.WebApi.Routes.IHttpRouteProvider], Coevery.Mvc.Routes.IRoutePublisher, System.Collections.Generic.IEnumerable`1[Coevery.Mvc.ModelBinders.IModelBinderProvider], Coevery.Mvc.ModelBinders.IModelBinderPublisher, Coevery.Tasks.ISweepGenerator)'.

**Comment by lfurzewaddock** [comment]
Hi ommvoice,

Unfortunately, I don't believe the original developers are supporting this project any longer.

I've forked it here: https://github.com/lfurzewaddock/Coevery-Orchard

My fork runs pretty well, but has less features as I forked it before the original developers made it difficult to maintain the underlying content management system: Orchard CMS http://orchardproject.net/

See this post for a little more info: http://orchard.codeplex.com/discussions/528259

This project has a lot of potential - can you help to resurrect it?

**Comment by ommvoice** [comment]
Thanks for the reply,

Actually I am more interested in Coevery, That is with angularJs.
http://coeverydemo.novasoftware.cn/#/

I cant not run this. any help will be much appreciated.

**Comment by lfurzewaddock** [comment]
Yes, Coevery is built on top of Orchard, extended by several plugin modules.

**Comment by hgiachetti** [comment]
Hi everyone, I am interested to rescue the project because there aren't .NET CRM open source.
I am beginning to understand the code and arquitecture.

I wil appreciate any help.

**Comment by vicpon** [comment]
Yes there are
http://www.splendidcrm.com/

**Comment by hgiachetti** [comment]
Thank you wujitouch for the answer!
I am going to try splendid crm but I saw the community version has less modules.
